### DISCUSSION TITLE
Demo Installation Help

### DISCUSSION THREAD

**Discussion by jplauer** [ckan/ckan#6233] [Q&A]
Hello everyone,

I am looking for more information on CKAN and would like to set up a test installation/look at the backend and play around with it.

The website https://demo.ckan.org/ is nice but does not offer access credentials for the backend. Does anyone have these credentials?

I also tried to install the latest ckan/postgresql/solr containers from Docker on my Qnap Container Station. However, the ckan container cannot reach the required port of the PostgreSQL server container with the error message "db:5432 - no response". I would also greatly appreciate your help with troubleshooting this issue.

Thanks,
Jan-Philipp

**Comment by anuveyatsu** [comment]
@jplauer if you need an account for demo site, we can provide you on request via this form:

https://www.datopian.com/contact/

**Reply by jplauer** [reply]
Thank you!

Have you received the filled in form I sent?

**Reply by monikappv** [reply]
Hi @jplauer. No, we haven't. Could you please either re-send your request or provide me with your e-mail address? Thanks.
